src/vue/components/panel-compact.vue

```diff
@@ -434,23 +321,45 @@ export default {
 
     showWordList () {
       return this.$store.state.app.wordListUpdated && this.app.wordlistC && Object.keys(this.app.wordlistC.wordLists) && Object.keys(this.app.wordlistC.wordLists).length > 0
+    },
+
+    formattedShortDefinitions () {
+      let definitions = []
+      if (this.$store.state.app.defDataReady && this.$store.state.app.homonym) {
+        for (const lexeme of this.$store.state.app.homonym.lexemes) {
+          if (lexeme.meaning.shortDefs.length > 0) {
+            definitions.push(...lexeme.meaning.shortDefs)
+          } else if (Object.entries(lexeme.lemma.features).length > 0) {
+            definitions.push(new Definition('No definition found.', 'en-US', 'text/plain', lexeme.lemma.word))
```

Comment:
please move this string to the messages file

Reply:
Done!